apps/web/src/pages/templates/workflow/ReplyCallback.tsx

```diff
@@ -1,17 +1,17 @@
 import styled from '@emotion/styled';
 import { Controller, useFormContext } from 'react-hook-form';
+import { useNavigate } from 'react-router-dom';
+import { Grid } from '@mantine/core';
+
 import { Input, Switch, Text } from '../../../design-system';
 import { useEnvController } from '../../../store/useEnvController';
 import { When } from '../../../components/utils/When';
-import React from 'react';
-import { useNavigate } from 'react-router-dom';
 import { DoubleArrowRight } from '../../../design-system/icons/arrows/CircleArrowRight';
-import { Grid } from '@mantine/core';
 
 export const ReplyCallback = ({ control, index, errors }) => {
   const { environment } = useEnvController();
-  const { getValues } = useFormContext();
-  const replyCallbackActive = getValues(`steps.${index}.replyCallback.active`);
```

Comment:
`getValues` only have an acces to the value during the render, but it doesn't listen to the value changes